### ISSUE TITLE
Is there any Sandbox , for testing all things ???

### ISSUE THREAD

**Issue by Govind13100** [nothingisdead/npm-kraken-api#59]
Hey there, 

I need help for sandbox, i have google but i didn't found any sandbox , can you please let me know that is there any sandbox available out ????

**Comment by CryptoMeerkat** [comment]
Nope, but you can pass in a validate flag to the API. For more information, refer to the Kraken API help pages. https://www.kraken.com/help/api

**Comment by shinlabs** [comment]
Found it under FAQ for API section : https://support.kraken.com/hc/en-us/articles/360000919926-Does-Kraken-offer-an-API-test-environment-
In short, when flag `validate: true` (in payload I guess) is present in api calls, kraken don't place the order, but answers with a json `{"error":[],"result":{"descr":{"order":"buy 5000.00000000 XDGUSD @ market"}}}`
